### DISCUSSION TITLE
Logging

### DISCUSSION THREAD

**Discussion by DariuszMrozik** [koechlm/Vault-Job-Processor---iLogic-Extension#21] [Q&A] [answered]
Hello,

I'm trying to log some information by inventor iLogic logger ( Logger.Debug("Some text")). I can run my rule. I have a log level set in addin interface. I have set path to log file. I have no file with log. Should I create some extra file for logs?

**Comment by DariuszMrozik** [comment, marked as answer]
I get error connected with log file. Maybe it's connected:
![image](https://user-images.githubusercontent.com/61918020/128088316-7cfcb805-8375-420a-a9ab-cdf4a07343cc.png)

**Reply by koechlm** [reply]
Hi Dariusz, which codebase are you working on? Or did you install the latest release setup? In case you compiled your own derivate I assume that you might have missed step 6 of the readme. In case you run on the latest installer, then you should check that the directory given is accessible. You do not need to create a logfile on your own upfront. The job creates one using the file name syntax: job.Id + "_" + mFile.Name + "_iLogicSampleJob.log". So, I suggest check the Readme bullet 6) first and check that the logging configuration is copied to the InventorServer bin dir.

**Reply by DariuszMrozik** [reply]
I have older version on code, because I would like to do tests on 2021 version of Vault. Step 6 of readme helps. Thank you. I have another issue. File named Test.IPT not working for me. 

If extension is write with capital letters there will be a problem. Could you confirm that? 

Is there an opportunity to do newest version of code to version 2021?

**Reply by koechlm** [reply]
You could take the latest code-base 2021 from this branch: https://github.com/koechlm/Vault-Job-Processor---iLogic-Extension/tree/2021.1.0; or take the latest while referencing 2021 SDK.
I neither could confirm nor reject the capital letter issue; I never had this but I surely do not handle it in the code. Could be a future improvement. I added an issue to the backlog: https://github.com/koechlm/Vault-Job-Processor---iLogic-Extension/issues/22
